### PR TITLE
#5 correct layout

### PR DESCRIPTION
**Pull request by trust2065**
First of all, I'm intend to use provide css and correct layout.
But it seems not right.

image

### CODE REVIEW

**Comment by josephj** [comment]
Please use Google Fonts. 
https://fonts.google.com/specimen/Open+Sans

**Comment by josephj** [comment]
I don't have the access to see the Inspect part. The CSS from the Invision is only for reference.

File: package.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "email": "trust2065@gmail.com",
   "license": "UNLICENSED",
   "scripts": {
-    "start": "set NODE_ENV=development&&webpack --mode=development --hot",
+    "start": "set NODE_ENV=production&&webpack --mode=development",
+    "dev": "set NODE_ENV=development&&webpack --mode=development --hot",
```
Comment (josephj) [inline package.json]: `yarn start` should have everything develop needs. It might not be very useful to make 2 different commands.

File: index.html [hunk 2]
```diff
@@ -134,19 +134,19 @@ <h2 class="textBox__head textBox__head--md textBox__head--white">
...
               />
             </div>
             <div class="textBox">
-              <h2 class="textBox__head textBox__head--md textBox__head--white">
+              <h2 class="textBox__head textBox__head--md textBox__head--grey">
                 Taking Taste Further
               </h2>
               <p
-                class="textBox__desc textBox__desc--sm textBox__desc--grey textBox__desc--maxL-3"
+                class="textBox__desc textBox__desc--sm textBox__desc--lightgrey textBox__desc--maxL-3"
```
Comment (josephj) [inline index.html]: Are you going to switch to bootstrap? The main benefit is that I don't need to understand your CSS naming strategy. And you CSS code will be more feature focused. You can do this in another pull request. Just want to know your plan.

File: src/style.scss [hunk 3]
```diff
@@ -155,10 +190,12 @@ body {
   }
   &__link {
     border-bottom: 2px solid #cf1430;
-    color: white;
     padding-bottom: 6px;
     text-decoration: none;
     text-transform: uppercase;
+
+    font-size: 16px;
+    font-weight: 600;
```
Comment (josephj) [inline src/style.scss]: 200 lines source code file is big nowadays. Better to use a CSS framework and split to different modules when it's necessary.

**Comment by trust2065** [comment]
> I don't have the access to see the Inspect part. The CSS from the Invision is only for reference.

This is what I expect to hear :D
Thank you for clean up my hesitation.

File: index.html (hunk 2 above)
Comment (trust2065) [reply index.html]: Yes I'm going to.
The purpose of this pull request is to know what I can get from invision, so I can go forward without hesitation.

I won't merge this PR :)
